File: spring-java-html/app/src/main/java/org/example/ServerApplication.java
package org.example;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;

@SpringBootApplication
@EnableConfigurationProperties(PortOneSecretProperties.class)
public class ServerApplication {
    public static void main(String[] args) {
        SpringApplication.run(ServerApplication.class, args);
    }
}

File: spring-html/app/bin/main/org/example/ServerApplication.kt
package org.example

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableConfigurationProperties(PortOneSecretProperties::class)
open class ServerApplication

fun main(args: Array<String>) {
    runApplication<ServerApplication>(*args)
}
